.github/workflows/build.yml

```diff
@@ -595,6 +608,9 @@ jobs:
           name: docs
           path: tmp/docs/
 
+      - name: Build mkdocs
+        run: pnpm build:mkdocs --build=false
```

Comment:
Is that `--build=false` bit correct?

It almost looks like you're building something, without actually running a (full) build?

Reply:
yes, it's skipping another docs generation which would lead to rate limiting on github api. See other conversation.